metricbeat/module/mysql/status/status.go

```diff
@@ -45,8 +45,9 @@ type MetricSet struct {
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	// Unpack additional configuration options.
 	config := struct {
-		Username string `config:"username"`
-		Password string `config:"password"`
+		Hosts    []string `config:"hosts"    validate:"nonzero,required"`
```

Comment:
That was only added for validation of Hosts? That would probably something that should be done on the module level as it applies to all mysql metricset (yes, there is currently only one) :-)